config/puma.rb

```diff
@@ -1,28 +1,30 @@
+# frozen_string_literal: true
+
 # Puma can serve each request in a thread from an internal thread pool.
 # The `threads` method setting takes two numbers: a minimum and maximum.
 # Any libraries that use thread pools should be configured to match
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
 # and maximum; this matches the default thread size of Active Record.
 #
-max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
-min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
+max_threads_count = ENV.fetch('RAILS_MAX_THREADS', 5)
```

Comment:
@olegbrest1 please fix GitHub Actions / fasterer here:

max_threads_count = ENV.fetch('RAILS_MAX_THREADS') { 5 }

If Rubocop will complain on it we need to add Rubocop settings to comply with GitHub Actions / fasterer